apps/full-site-editing/full-site-editing-plugin/block-patterns/patterns/images.php

```diff
@@ -0,0 +1,71 @@
+<!-- /wp:image --></div>
+<!-- /wp:jetpack/layout-grid-column --></div>
+<!-- /wp:jetpack/layout-grid --></div>
+<!-- /wp:jetpack/layout-grid-column -->
+
+<!-- wp:jetpack/layout-grid-column -->
+<div class="wp-block-jetpack-layout-grid-column wp-block-jetpack-layout-grid__padding-none"><!-- wp:paragraph {"className":"margin-bottom-none","style":{"typography":{"fontSize":14}}} -->
+<p class="margin-bottom-none" style="font-size:14px">%1$s</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:heading {"level":3,"className":"margin-top-none","style":{"typography":{"fontSize":24,"lineHeight":"1.2"}}} -->
+<h3 class="margin-top-none" style="line-height:1.2;font-size:24px">%2$s</h3>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph {"style":{"typography":{"fontSize":14}}} -->
+<p style="font-size:14px">%3$s </p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph {"style":{"typography":{"fontSize":14}}} -->
+<p style="font-size:14px">%4$s</p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:jetpack/layout-grid-column --></div>
+<!-- /wp:jetpack/layout-grid -->
+';
+
+return array(
+	'__file'     => 'wp_block',
+	'title'      => esc_html__( 'Images', 'full-site-editing' ),
+	'categories' => array( 'gallery', 'images', 'text' ),
+	'content'    => sprintf(
+		$markup,
+		esc_html__( 'Vol.3', 'full-site-editing' ),
+		esc_html__( '5am, the Beach', 'full-site-editing' ),
+		esc_html__( 'As long as I was on the path I walked hard, but when I came to the black beach I had to run. For the tide was now nearly flowed; and to get through with my powder dry between the surf and the steep hill, took all the quickness I possessed. As it was, even, the wash caught me to the knees, and I came near falling on a stone. All this time the hurry I was in, and the free air and smell of the sea, kept my spirits lively; but when I was once in the bush and began to climb the path I took it easier.', 'full-site-editing' ),
+		esc_html__( 'The fearsomeness of the wood had been a good bit rubbed off for me by Master Case’s banjo-strings and graven images, yet I thought it was a dreary walk, and guessed, when the disciples went up there, they must be badly scared. The light of the lantern, striking among all these trunks and forked branches and twisted rope-ends of lianas, made the whole place, or all that you could see of it, a kind of a puzzle of turning shadows. They came to meet you, solid and quick like giants, and then span off and vanished; they hove up over your head like clubs, and flew away into the night like birds. The floor of the bush glimmered with dead wood, the way the match-box used to shine after you had struck a lucifer. Big, cold drops fell on me from the branches overhead like sweat. There was no wind to mention; only a little icy breath of a land-breeze that stirred nothing; and the harps were silent.', 'full-site-editing' )		
```

Comment:
This is throwing a couple of PHPCS whitespace errors:

```
 69 | ERROR | [x] Spaces must be used for mid-line alignment; tabs
    |       |     are not allowed
    |       |     (WordPress.WhiteSpace.DisallowInlineTabs.NonIndentTabsUsed)
 69 | ERROR | [x] Whitespace found at end of line
    |       |     (Squiz.WhiteSpace.SuperfluousWhitespace.EndLine)
```

Reply:
Derp. Will fix.